crates/conformance-test-utils/src/utils.rs

```diff
@@ -0,0 +1,58 @@
+use starknet::core::types::FieldElement;
+use starknet::core::utils::get_contract_address;
+
+/// Computes the StarkNet address of EVM address given the Kakarot address and Account Proxy Class
+/// Hash. Uses the same logic as get_contract_address with EVM as salt.
+pub fn compute_starknet_address(
+    kakarot_address: FieldElement,
+    account_proxy_class_hash: FieldElement,
+    evm_address: FieldElement,
+) -> FieldElement {
+    let constructor_calldata: Vec<FieldElement> = Vec::new();
+    get_contract_address(evm_address, account_proxy_class_hash, &constructor_calldata, kakarot_address)
+}
```

Comment:
This uses the `evm_address` as salt and computes the starknet address using `get_contract_address` which does a pedersen hash on elements recursively